### ISSUE TITLE
Pretrained Model Results

### ISSUE THREAD

**Issue by emreunel** [marshuang80/PENet#14]
Hello,

My name is Emre Ünel, and I would like to use your PeNet model to test for embolism detection. To do this, I downloaded the GitHub link you published in your article "PENet—a scalable deep-learning model for automated diagnosis of pulmonary embolism using volumetric CT imaging" and installed the pre-trained model on my computer. I am currently trying to test the model with the Stanford test set that you used in your article. There were a total of 190 npy files in your test set, and I loaded your model into my code. I created 24 axial slices with a 23-slice overlap. As you shared in your article and code, I resized the images to 208x208 and 224x224, as you did, and then cropped the central 192x192 portion. I performed min-max normalization using minimum (-1000) and maximum(900) values, clipped the range to 0-1, and then subtracted mean value (0.15897) from the image. However, when I look at the detection results, I am not getting the same results as those published in your article. Is the code or pre-trained model up-to-date, or could I have made a mistake somewhere? Thank you for your help.

Best regards,
Zeki Emre Ünel.
